physical/raft/raft.go

```diff
@@ -316,6 +316,24 @@ func NewRaftBackend(conf map[string]string, logger log.Logger) (physical.Backend
 	}, nil
 }
 
+// Close is used to gracefully close all file resources.  N.B. This method
+// should only be called if you are sure the RaftBackend will never be used
+// again.
+func (b *RaftBackend) Close() error {
+	b.l.Lock()
+	defer b.l.Unlock()
+
+	if err := b.fsm.db.Close(); err != nil {
+		return err
+	}
+
+	if err := b.stableStore.(*raftboltdb.BoltStore).Close(); err != nil {
```

Comment:
Is `stableStore` always guaranteed to be a reference to a `raftboltdb.BoltStore`?

Reply:
Yes, except if you happen to be in devMode, which is currently hard wired never happen:  https://github.com/hashicorp/vault/blob/master/physical/raft/raft.go#L229-L234